test/helpers/position_assertions.cc

```diff
@@ -763,6 +769,39 @@ string TypeAssertion::toString() const {
     return fmt::format("type: {}", symbol);
 }
 
+FileDefAssertion::FileDefAssertion(std::string_view filename, std::unique_ptr<Range> &range, int assertionLine,
+                                   std::string targetFilename, int targetLine, int targetColumn)
+    : BaseDefAssertion(filename, range, assertionLine, ""sv), targetFilename(targetFilename), targetLine(targetLine),
+      targetColumn(targetColumn) {}
+
+std::shared_ptr<FileDefAssertion> FileDefAssertion::make(std::string_view filename, std::unique_ptr<Range> &range,
+                                                         int assertionLine, std::string_view assertionContents,
+                                                         std::string_view assertionType) {
+    vector<string_view> split = absl::StrSplit(assertionContents, ' ');
+    CHECK_EQ(split.size(), 3);
+    {
+        INFO(fmt::format("Invalid file-def assertion; multiple words found:\n{}\nFile definition assertions should be "
+                         "of the form:\n# [^*] file-def: file-path line column",
+                         assertionContents));
+    }
+    auto file_path = split[0];
+    auto line = stoi(string(split[1]));
+    auto col = stoi(string(split[2]));
```

Comment:
Will these ever be anything but `0`? This assertion seems overly general.

Reply:
Ultimately I was thinking of improving this functionality to jump to the first definition in the file (e.g. the first module/class definition) hence why I would need a specific line/column

Reply:
In that case, a landing label like we do for other jump-to assertions seems nice. that avoids the brittle annotations problem, where adding whitespace would inadvertently break an assertion